tests/regression/76-memleak/16-no-mem-leak-thread-exit-main.c

```diff
@@ -0,0 +1,22 @@
+//PARAM: --set ana.malloc.unique_address_count 1 --set ana.activated[+] memLeak --set ana.activated[+] thread
+#include <stdlib.h>
+#include <pthread.h>
+
+int *m1;
+
+void *f1(void *arg) {
+    m1 = malloc(sizeof(int));
+    while (1);
+    return NULL;
+}
+
+int main(int argc, char const *argv[]) {
+    pthread_t t1;
+    pthread_create(&t1, NULL, f1, NULL);
+
+    pthread_exit(NULL);
+
+    // A pthread_join called in main will wait for other threads to finish
+    // Therefore, no memory leak here
+    return 0; // NOWARN
```

Comment:
This is dead code, so of course it's `NOWARN`. But shouldn't the `NOWARN` be on the `pthread_exit` before?